var/spack/repos/builtin/packages/topaz/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Topaz(PythonPackage):
+    """topaz: Pipeline for particle picking in cryo-electron microscopy images using
+    convolutional neural networks trained from positive and unlabeled examples. Also
+    featuring micrograph and tomogram denoising with DNNs."""
+
+    homepage = "https://topaz-em.readthedocs.io/"
+    git = "https://github.com/tbepler/topaz.git"
+
+    version("0.2.5", tag="v0.2.5")
```

Comment:
This package is on Pypi, where you than also have a proper checksum:
https://pypi.org/project/topaz-em/

Reply:
So there is! Switched over in a new push ...